collect_app/src/main/java/org/odk/collect/android/application/Collect.java

```diff
@@ -272,5 +281,20 @@ synchronized public Tracker getDefaultTracker() {
         }
         return mTracker;
     }
+    private static class CrashReportingTree extends Timber.Tree {
+        @Override
+        protected void log(int priority, String tag, String message, Throwable t) {
+            if (priority == Log.VERBOSE || priority == Log.DEBUG || priority == Log.INFO) {
+                return;
+            }
+
+            //Changed from Log.ERROR to priority below.
```

Comment:
Great! But now I don't think this comment is needed. If you want to point something out to a reviewer you can add one of these kinds of GitHub comments that don't go in the code. 😊